packages/dialogs/src/RPA/Dialogs/library.py

```diff
@@ -16,7 +16,12 @@
 
 @library(scope="GLOBAL", doc_format="REST", auto_keywords=False)
 class Dialogs:
-    """The `Dialogs` library provides a way to display information to a user
+    """**NOTICE:** The `Dialogs` library is now marked for deprecation.
+    The library will be replaced by `Assistant`. In order to get more
+    information and check the changes and the migration guide, please
```

Comment:
```suggestion
    information about the changes and the migration guide, please
```
to avoid doubling "and" & "check".